adminsortable2/admin.py

```diff
@@ -97,9 +97,9 @@ def change_list_template(self):
         opts = self.model._meta
         app_label = opts.app_label
         return [
-            Path('adminsortable2') / Path(app_label) / Path(opts.model_name) / Path('change_list.html'),
-            Path('adminsortable2') / Path(app_label) / Path('change_list.html'),
-            Path('adminsortable2/change_list.html'),
+            os.path.join('adminsortable2', app_label, opts.model_name, 'change_list.html'),
```

Comment:
you can always convert a `Path` object to a string by wrapping it into `str(…)`, which is the preferred way.